src/firebase/firebase.tsx

```diff
@@ -12,8 +13,9 @@ const firebaseConfig = {
     measurementId: process.env.REACT_APP_MEASUREMENT_ID,
 }
 
-firebase.initializeApp(firebaseConfig)
-
+const firebaseInstance = firebase.initializeApp(firebaseConfig)
+export const firebaseAuth = firebase.auth
+export const authInstance = firebaseInstance.auth()
```

Comment:
did you mean to use `firebaseAuth` variable created above here ?

Reply:
no, we could have many Firebase config within a single project, this specificly require the firebaseInstance auth rather than the global defined one. In our case, it result in the same behavior though.